main.py

```diff
@@ -17,8 +17,8 @@ def getFileText(name):
 
 while True and not compileFile:
     if len(sys.argv) < 2:
-        text = input('lazy > ')
-        if text.strip() == "quit()": break
+        text = input('lazy > ').upper()
```

Comment:
This is not a proper fix for these issues.
You need to change keywords in the code to make this case insensitive.

Adding .upper() is just like adding another layer of string conversion,

Reply:
@OrignalLazyCoder  If I change the keywords to lower case , then if someone enter upper case then the interpreter will show error , so it will not make code insensitive. So,I think adding a another layer is optimal solution.

Reply:
@OrignalLazyCoder  I have now commited the code in `dev` branch , you may close this PR now. And guide me futher in that PR.

Reply:
Please look into this issue properly. You can change the keywords and you might need to make changes in the Interpreter, Value, and Parser file.